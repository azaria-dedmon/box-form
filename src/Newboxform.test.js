import { render, screen } from '@testing-library/react';
import NewBoxForm from './Newboxform';

it("renders without crashing", function() {
    render(<NewBoxForm />);
  });
  
  it("matches snapshot", function() {
    const { asFragment } = render(<NewBoxForm />);
    expect(asFragment()).toMatchSnapshot();
  });