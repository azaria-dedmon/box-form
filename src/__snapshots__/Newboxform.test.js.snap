// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <form>
    <label
      for="width"
    >
      Width
    </label>
    <input
      id="width"
      name="width"
      placeholder="Box width"
      type="number"
      value="0"
    />
    <label
      for="height"
    >
      Height
    </label>
    <input
      id="height"
      name="height"
      placeholder="Box height"
      type="number"
      value="0"
    />
    <label
      for="backgroundcolor"
    >
      Background Color
    </label>
    <input
      id="backgroundcolor"
      name="backgroundColor"
      placeholder="Box background color"
      type="text"
      value=""
    />
    <button>
      Add box
    </button>
  </form>
</DocumentFragment>
`;
